highfidelity/hifi
# Preload collision sounds

When an entity is added or changed with a collisionSoundURL, cache it so that it is ready to play on the first hit.

To test all sounds now:
1. You should be able to remove top-level occurrences of SoundCache.getSound() that are there only for side-effect. For example in dice.js, remove SoundCache.getSound("http://s3.amazonaws.com/hifi-public/sounds/dice/diceCollide.wav".  N.B.: If you ALSO clear your disk cache, it may be that you won't get the sound over the network fast enough!
2. From earlier checkins, you should be able to add a collisionSoundURL on non-moving items, such as the bumpers in the air hockey table. With this checkin, those should work without having pre-cache them with SoundCache.getSound().
